INSTALLATION.md

```diff
@@ -288,6 +289,17 @@ We have created a setup script to make configuring Talawa-API easier.
 ```
 npm run setup
 ```
+## Install Using Devpod
+This guide provides a step-by-step guide to setting up a Talawa-Api server using Devpod.
+
+Follow these steps:
+
+1. Install the Devpod GUI application or Devpod CLI. [Learn more](https://devpod.sh/docs/getting-started/install)
```

Comment:
Is this method independent of installing node.js and typescript as prerequisites? If it is, then it should be its own section. Think of the end user, not everyone is familiar with this technology.

Reply:
yes it is independent of installing node. js and ts , I didn't get with the point it should be its own section , could you clearify it more ?

Reply:
We have a section in the table of contents for installing prerequisites. If we need to install different prerequisites depending on whether we use devcontainer or not, then we need the prerequisites section to be split in two for each scenario.
 
![image](https://github.com/user-attachments/assets/0cf0b42e-f2b3-4e99-b31d-b2c6f30a8bc2)